includes/Services/PluginInstaller.php

```diff
@@ -162,6 +162,9 @@ public static function install_from_wordpress( $plugin, $activate ) {
 	 * @return \WP_Error|\WP_REST_Response
 	 */
 	public static function install_premium_plugin( $plugin, $provider, $activate ) {
```

Comment:
Do we default `$activate` to true anywhere? In 99.999999% of all cases, we want to activate the plugin after installing. It would reduce the number of attributes we need to set.

We should probably use clearer variable naming like `$shouldActivate`

Reply:
Yea, good call. That threw me off for a minute - especially since we're talking about activating plugins AND licenses here. I'll update real quick.